packages/nextjs-template/package.json

```diff
@@ -54,7 +55,8 @@
     "@playwright/test": "^1.18.1",
     "@types/fs-extra": "^9.0.12",
     "@types/luxon": "^2.3.1",
-    "@types/react": "17.0.39",
+    "@types/react": "18.0.20",
```

Comment:
we've reverted back to 17.0.39 in mainline because 18 introduced type errors. can you revert this?

Reply:
Reverted